### PR TITLE
Bugfix for PSR4 support

### PR DESCRIPTION
**Pull request by shandyDev**
Missing fix which already have in RestServiceModel.
May be need cover under tests this?
Ping @weierophinney

### CODE REVIEW

File: src/Model/RpcServiceModel.php [hunk 1]
```diff
@@ -763,4 +763,19 @@ public function createMediaType()
             $this->moduleEntity->getLatestVersion()
         );
     }
+
+     /**
```
Comment (webimpress) [inline src/Model/RpcServiceModel.php]: Please remove one space at the beginning of this line.
Comment should look like:
```
/**
 * ...
```

Comment (shandyDev) [inline src/Model/RpcServiceModel.php]: I see Travis status. Already fix it. )

Comment (webimpress) [inline src/Model/RpcServiceModel.php]: You've fixed whitespaces in line above, your comment starts
```
/**
*
```
but it should be:
```
/**
 *
```

so you need to have one space less in line 767 `/**` (you have there 5 spaces, should be just 4).

This is not defined in CS checks. Please have a look how others comments looks.

**Comment by shandyDev** [comment]
It seems Travis has failed:

> The command "eval curl https://raw.githubusercontent.com/travis-ci/casher/production/bin/casher  -L -o $CASHER_DIR/bin/casher -s --fail" failed. Retrying, 3 of 3.

Please, rebuild it.

File: src/Model/RpcServiceModel.php (hunk 1 above)
Comment (shandyDev) [inline src/Model/RpcServiceModel.php]: Sorry, misunderstanding. Fixed.

**Comment by shandyDev** [comment]
@weierophinney When can expect new release with this PR? 1.5.10?

**Comment by Techworker** [comment]
I would like to have this merged too.

**Comment by weierophinney** [comment]
Thanks, @shandyDev

**Pull request merged**